services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/oauth/ClaimsJsonUtilTest.java

```diff
@@ -104,9 +108,15 @@ public void testAddAndExtractScopeAndClaims() throws Exception {
 		// method under test
 		ClaimsJsonUtil.addAccessClaims(scopes, oidcClaims, claims);
 		
+		// we need to simulate the round trip to a serialized form and back
+		String token = Jwts.builder().setClaims(claims).compact();
+		Claims parsedClaims = Jwts.parser().parseClaimsJwt(token).getBody();
```

Comment:
add an assertion that the serialized JSON body is literally equal to an expected string